libkbfs/data_types.go

```diff
@@ -960,3 +960,9 @@ func SessionInfoFromProtocol(session keybase1.Session) (SessionInfo, error) {
 		VerifyingKey:   verifyingKey,
 	}, nil
 }
+
+// NodeMetadata has metadata about a node needed for higher level operations.
+type NodeMetadata struct {
+	LastWriterAccordingToTLFLastWriter libkb.NormalizedUsername
```

Comment:
I like the intent here, but I'm not sure we should bake the term "TLF" into this data structure.  I think "LastWriterUnverified" (along with a comment saying why it's unverified, plus documentation we'll add on the website too) is clear enough.